brave/brave-core
# Adds contribution info to tipping banner

Fixes brave/brave-browser#6744

Some of the spacing from the mock-up had to be tightened else the banner would have become vertically over-extended

<img width="1162" alt="Screen Shot 2019-11-14 at 1 34 59 PM" src="https://user-images.githubusercontent.com/8732757/68894217-143be080-06e4-11ea-9921-d3660aa15be3.png">
<img width="1161" alt="Screen Shot 2019-11-14 at 1 34 41 PM" src="https://user-images.githubusercontent.com/8732757/68894218-14d47700-06e4-11ea-8e6e-1edb8060a0e6.png">


## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Enable Rewards
2. Claim a grant
3. Navigate to 3zsistemi.si
4. Send a tip
5. Confirm the tip amount is shown in the confirmation
6. Close banner
7. Set a monthly contribution via the panel for 3zsistemi.si
8. Confirm the monthly amount and next contribution date is shown in the confirmation

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
